src/dd-doctor.php

```diff
@@ -120,7 +120,7 @@ function env($key)
 function check_opcache()
 {
     if (!function_exists('opcache_get_configuration')) {
-        render('Opcache enabled:', 'NO');
+        return render('Opcache enabled:', 'NO');
```

Comment:
Thanks for the PR! I prefer it this style since `render` doesn't return anything:
```suggestion
       render('Opcache enabled:', 'NO');
       return;
```